e2e/libs/k8s/k8s.py

```diff
@@ -32,6 +32,12 @@ async def restart_kubelet(node_name, downtime_in_sec=10):
 
     delete_pod(pod_name)
 
+def get_longhorn_node_condition_ready_status(node_name):
```

Comment:
I am thinking if we could make this a more generic function.
```
def get_longhorn_node_condition_status(node_name, type):
```
What do you think? I am also fine with the current.

Reply:
Updated!